packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -258,3 +284,11 @@ class EventDetailsTabNavTab extends PTabNavTab {
 
   final EventDetailsTabType type;
 }
+
+class EmptyCpuProfileException implements Exception {
```

Comment:
I would recommend avoiding defining a bunch of custom exception classes. If we want an exception type that indicates VM service errors with a message that would be fine.
In general for this code you would be fine catching any error that indicated an anticipated vm service exception not just the specific one.